api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -521,31 +523,31 @@ def touch_tip(
     ) -> InstrumentContext:
         """
         Touch the pipette tip to the sides of a well, with the intent of
-        removing left-over droplets
+        removing left-over droplets. See also, :ref:`touch-tip`.
```

Comment:
closed compound. and the comma thing.
```suggestion
        removing leftover droplets. See also :ref:`touch-tip`.
```